test/integration/wdio.conf.js

```diff
@@ -196,21 +193,19 @@ function makeCapabilities(config) {
         case 'chromedriver':
             return (() => {
                 const args = [
-                    '--disable-gpu',
-                    '--no-sandbox',
+                    'headless',
+                    'disable-gpu',
                     `window-size=${config.WIDTH},${config.HEIGHT}`,
                 ];
-                if (config.HEADLESS === 't') {
-                    args.push('--headless');
-                }
                 return {
                     browserName: 'chrome',
+                    browserVersion: '128',
```

Comment:
latest version of Chrome. Not sure yet if we can add a spread of these (or if we need to, I think our policy is to stick to the latest).

That should probably get configured elsewhere rather than as a magic number, but... later.